src/pages/Swap/index.tsx

```diff
@@ -677,15 +677,15 @@ export default function Swap() {
                         }
                       >
                         <AutoRow justify="space-between" style={{ flexWrap: 'nowrap' }} height="20px">
-                          <span style={{ display: 'flex', alignItems: 'center' }}>
+                          <Text display="flex" fontSize={'16px'} justifyContent={'center'} width={'100%'}>
```

Comment:
Is there a pattern in the codebase for using plain strings vs wrapping them with curlies? I think I've seen both in the codebase. Either way, it would be great to maintain consistency going forward!

Reply:
@aballerr, could you clarify which one would keep things consistent? Using styled-component `Text` from rebass or `ThemedText`. Would make the changes accordingly. Thanks

Reply:
Thanks @grabbou, looks like the ThemedText pattern is prominent in the codebase, have changed to that. Could you review again? Thanks
cc: @aballerr

Reply:
Also, ended up using `ThemedText.SubHeader` over `ThemedText.BodyPrimary` because of the fontWeight, ThemedText.BodyPrimary with fontWeight 400 looked odd in compared to the `Swap` text below. Hope this works.

### With ThemedText.BodyPrimary
![Screenshot 2022-11-15 at 9 07 10 AM](https://user-images.githubusercontent.com/44068102/201821014-203b8064-22b0-4463-ac5a-f8e2c7ef54e5.png)

### With ThemedText.SubHeader
![Screenshot 2022-11-12 at 12 35 13 PM](https://user-images.githubusercontent.com/44068102/201821276-147aefc4-f6bc-4e51-b289-b07a4f234eb4.png)